Floyd's_Cycle_Detection_Algorithm/Floyd's_Cycle_Detection.cpp

```diff
@@ -0,0 +1,51 @@
+#include<bits/stdc++.h>
+using namespace std;
+ 
+// Structure of Linked List node
+struct Node 
+{ 
+    int data; 
+    Node* next; 
+};
+Node *head = NULL;
+
+void insert(int value) 
+{ 
+    // Create a new node and adjusting links
+    Node* temp = new Node;
+    temp->data = value; 
+    temp->next = head; 
```

Comment:
You have values array as 10, 20, 30, 40. But wouldn't this function create the linked list in reverse manner? (40 -> 30 -> 20 -> 10). Can you maintain the same order as the original array?

Reply:
I have made the required changes. Please review.